src/components/filters/FilterList/index.jsx

```diff
@@ -6,8 +6,11 @@ import './FilterList.css';
 class FilterList extends React.Component {
   constructor(props) {
     super(props);
-    const initialFilterStatus = props.sections
+    let initialFilterStatus = props.sections
       .map(() => ({}));
+    if (props.filterStatusFromParent && Object.keys(props.filterStatusFromParent).length > 0) {
```

Comment:
why not just `props.filterStatusFromParent.length > 0` since that props is an array?